### PR TITLE
Documentation: Clarifies how to assign a template to a default Post Type

### PR DESCRIPTION
**Pull request by youknowriad**
Everything is in the title

### CODE REVIEW

File: docs/templates.md [hunk 1]
```diff
@@ -68,3 +68,17 @@ Sometimes the intention might be to lock the template on the UI so that the bloc
 
 - `all` — prevents all operations.
 - `insert` — prevents inserting new blocks, but allows moving existing ones.
+
+## Default Post Types
```
Comment (mtias) [inline docs/templates.md]: Maybe "Existing Post Type"?

**Review by mtias — APPROVED** [review]

File: docs/templates.md (hunk 1 above)
Comment (youknowriad) [reply docs/templates.md]: I took the term from the Codex https://codex.wordpress.org/Post_Types might be good to match it.

Comment (mtias) [reply docs/templates.md]: Right, but I was thinking of cases where a CPT is registered elsewhere by another person.

Comment (youknowriad) [reply docs/templates.md]: Good point 👍

**Pull request merged**